source/pkcore/abilitypages/chi/paralyze.rst

```diff
@@ -0,0 +1,59 @@
+.. paralyze:
+############
+Paralyze
+############
+
+Description
+###########
+
+Paralyze is a Chi Ability.
+Paralyze allows the Chiblocker to paralyze entities for a short time period. GIF 
```

Comment:
Change to `Paralyze allows Chiblockers to stun entities for a short period of time during which they are completely helpless. GIF TO BE ADDED`